cherrypy/_cpdispatch.py

```diff
@@ -46,10 +48,12 @@ def kwargs(self):
 
     @kwargs.setter
     def kwargs(self, kwargs):
+        """Setter for named kwargs."""
```

Comment:
Methods/functions should usually start with a verb. The property getter is an exception, but I don't thing a setter is.

I'd go for something like "Set named keyword arguments on the current request object."

Reply:
```suggestion
        """Set the named request keyword arguments as :class:`dict`."""
```